docs/api/model_selection/importances.rst

```diff
@@ -107,10 +107,54 @@ Taking the mean of the importances may be undesirable for several reasons. For e
     X, y = data.data, data.target
 
     model = LogisticRegression(multi_class="auto", solver="liblinear")
-    viz = FeatureImportances(model, stack=True, relative=False)
+    viz = FeatureImportances(model, stack=True, relative=False, topn=3)
     viz.fit(X, y)
     viz.show()
 
+Top and Bottom Feature Importances
+----------------------------------
+
+It may be more illuminating to the feature engineering process to identify the most or least informative features. To view only the N most informative features in a graph, use ``topn`` with a postive integer. To view the N least important, use a negative integer.
+
+.. plot::
+    :context: close-figs
+    :alt: Coefficient importances for LASSO regression
+
+    from sklearn.linear_model import Lasso
+    from yellowbrick.datasets import load_concrete
+    from yellowbrick.model_selection import FeatureImportances
+
+    # Load the regression dataset
+    dataset = load_concrete(return_dataset=True)
+    X, y = dataset.to_data()
+
+    # Title case the feature for better display and create the visualizer
+    labels = list(map(lambda s: s.title(), dataset.meta['features']))
+    viz = FeatureImportances(Lasso(), labels=labels, relative=False, topn=3)
+
+    # Fit and show the feature importances
+    viz.fit(X, y)
+    viz.show()
+
+When repeating the previous concrete dataset graph using ``topn=3``, we see that the three most informative features are splast, cement, and water. Although water has a negative coefficent, it is nonetheless informative to reveal the negative correlation associated with the feature.
```

Comment:
Suggested edit:

Using `topn=3`, we can identify the three most informative features in the concrete dataset as `splast`, `cement`, and `water`. This approach to visualization may assist with _factor analysis_ - the study of how variables contribute to an overall model. Note that although `water` has a negative coefficient, it is the magnitude (absolute value) of the feature that matters since we are closely inspecting the negative correlation of water with strength. An alternative, `topn=-3` would reveal the least informative features in the model. This approach is useful to model tuning similar to [RFECV](https://www.scikit-yb.org/en/latest/api/model_selection/rfecv.html), but instead of automatically removing features, would allow you to identify the lowest-ranked features as they change in different model instantiations. In either case, if you have many features, using `topn` can significantly increase the visual and analytical capacity of your analysis. 


Note on the `RFECV` link, please use:

```
:doc:`RFECV <api/model_selection/rfecv>`
```

As the API link in rst, but you may have to ensure the path is relative if the docs do not build (the path given above is from the root).

Reply:
Thank you for the domain knowledge!